docs/non_containerized_NSFS.md

```diff
@@ -0,0 +1,62 @@
+# Non Containerized NSFS
+
+Running nsfs non containerized is useful for deploying in linux without depending on kubernetes, but requires some steps which are described next.
+
+
+## Build
+
+### Build options - 
+1. noobaa-core S3 Select is enabled. This requires boost shared objects to be present on the machine where noobaa is supposed to be installed. This feature can be disabled if not required (due to additional dependency) by running make rpm BUILD_S3SELECT=0.
+2. In order to build RPM packages for other architectures simply run make rpm CONTAINER_PLATFORM=linux/amd64 or make rpm CONTAINER_PLATFORM=linux/ppc64le
+
+Running the following command will result with an RPM file -
+```sh
+make rpm
+```
+
+
+## Install
+
+### Pre-requisites (S3 select enabled) - 
+1. wget - 
+
+```sh
+yum install wget
+```
+2. boost -
+```sh
+wget https://rpmfind.net/linux/centos/8-stream/AppStream/x86_64/os/Packages/boost-system-1.66.0-13.el8.x86_64.rpm
+wget https://rpmfind.net/linux/centos/8-stream/AppStream/x86_64/os/Packages/boost-thread-1.66.0-13.el8.x86_64.rpm
+rpm -i boost-thread-1.66.0-13.el8.x86_64.rpm
+rpm -i boost-system-1.66.0-13.el8.x86_64.rpm
+```
+
+### Install NooBaa RPM - 
+Download the RPM to the machine and install it by running the following command - 
```

Comment:
I think the command following this line would just install the RPM and not download it.

Reply:
@tangledbytes Yes, it's just installation for now because I don't know where will the RPM be published.
@nimrod-becker @guymguym We need to discuss what RPM download instructions will look like,
1. Where these RPMs are published?
3. Will the RPM be based on our regular version branches?
4. Will this RPM build be automated like DS/US builds?